.github/workflows/tests.yml

```
@@ -28,26 +28,26 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node_version }}
-      - run: npm i
-      - run: npm run test
+      - run: yarn
+      - run: yarn test
   integration:
     runs-on: ubuntu-latest
-    services:
-      ircd:
-        image: inspircd/inspircd-docker
-        env:
-          # Don't check domain against blacklists
-          INSP_ENABLE_DNSBL: no
-        ports:
-          - 6667:6667
     strategy:
       matrix:
         node_version: [16, 18]
-    steps:
+    steps:      
```

Remove white-spaces.
```suggestion
    steps:
```